src/App.js

```diff
@@ -0,0 +1,135 @@
+  margin-top: ${(props) => (props.isDetected ? "25vh" : 0)};
+  text-align: center;
+  font-size: 4rem;
+  color: white;
+  ${(props) => props.complex && complexMixin};
+  order: ${(props) => (props.isDetected ? 1 : 2)};
+  opacity: ${(props) => (props.isDetected ? 1 : 0)};
+`;
+
+const Logo = styled.img`
+  order: ${(props) => (props.isDetected ? 2 : 1)};
+  transform: ${(props) => (props.isDetected ? "scale(1)" : "scale(0.5)")};
+  margin-top: ${(props) => (props.isDetected ? 0 : "10vh")};
+`;
+
+function App() {
+  const [videoElement, setVideoElement] = React.useState(null);
+  const [isDetected, setIsDetected] = React.useState(false);
+
+  React.useEffect(() => {
+    setVideoElement(document.getElementById("video"));
+
+    if (videoElement === null) {
+      console.log("VideoElement doesn't exist");
+      return;
+    }
+    navigator.mediaDevices
+      .getUserMedia({
+        audio: false,
+        video: {
+          width: { ideal: 720 },
+          height: { ideal: 560 },
+        },
+      })
+      .then((stream) => {
+        videoElement.srcObject = stream;
+        startDetection();
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+
+    loadSsdMobilenetv1Model("/models")
+      .then(() => {
+        console.log("model loaded");
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  }, [videoElement]);
+
+  async function startDetection() {
+    const videoElement = document.getElementById("video");
+    setInterval(async () => {
+      const detection = await detectSingleFace(
+        videoElement,
+        new SsdMobilenetv1Options({ minConfidence: 0.1 })
+      );
+
+      if (detection == null) {
+        setIsDetected(false);
+        console.log("detection is null");
+        return;
+      }
+      if (detection._score > 0.5) {
+        setIsDetected(true);
+        console.log("Found face");
+      } else {
+        setIsDetected(false);
+        console.log("Didn´t find face");
+      }
+      console.log("Detection score: " + detection._score.toFixed(2));
+    }, 500);
+  }
+
+  return (
+    <div className="wrapper">
+      <Title isDetected={isDetected}>Välkommen till</Title>
+      <Logo
+        src={dynabyteLogo}
+        alt="logo"
+        width="200"
+        height="80"
+        isDetected={isDetected}
+      />
+      <video
+        style={{ opacity: 0 }}
+        id="video"
+        width="720"
+        height="560"
+        autoPlay={true}
+        muted={true}
+      ></video>
+      <footer>
+        <span>
+          Photo by{" "}
+          <a href="https://unsplash.com/@freetousesoundscom?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">
+            Free To Use Sounds
+          </a>{" "}
+          on{" "}
+          <a href="https://unsplash.com/s/photos/grass?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">
+            Unsplash
+          </a>
+        </span>
```

Comment:
Ska vi ha kvar detta? Går det kanske att ta bort helt? :)

Reply:
Om vi använder den här bilden, ska det vara kvar.

Reply:
Jag skapar en annan branch som ska merga in till POC-React-Javascript istället nu..

Reply:
Nej det går inte.. jag har uppdaterat POC-React-Javascript med den här PR.

Reply:
Ok, då är det ok för tillfället, vi får se om vi kanske ska ha en annan bild senare då, för tillfället ser det bra ut :)